packages/arb-bridge-peripherals/hardhat.config.ts

```diff
@@ -1,3 +1,34 @@
-import * as config from "arb-bridge-eth/hardhat.config"
-// use same config from bridge package
+import * as config from 'arb-bridge-eth/hardhat.config'
+import { task } from 'hardhat/config'
+import { initUpgrades } from 'arb-upgrades'
+
+require('@openzeppelin/hardhat-upgrades')
+
+task('deploy-logic-one', 'deploy one logic')
+  .addParam('contract', 'contract to deploy')
+  .setAction(async (args, hre) => {
+    const { contract } = args
```

Comment:
should we explicitly validate this input is valid? or do we get a clean type error when we try to use an incorrect input in `await deployLogic(contract)`

Reply:
getContractFactory throws a clean error if the contract name is unrecognized